modules/administration-guide/pages/configuring-the-administrative-user.adoc

```diff
@@ -0,0 +1,28 @@
+:_content-type: PROCEDURE
+:navtitle: Configuring the administrative user
+:keywords: administration-guide, admin
+:page-aliases:
+
+[id="configuring-the-administrative-user_{context}"]
+= Configuring the administrative user
+
+To execute actions that require administrative privileges on {prod-short} server, such as deleting user data, activate a user with administrative privileges.
```

Comment:
To execute actions that require administrative privileges on {prod-short} server, such as deleting user data, activate an `admin` user.